docker-compose.yaml

```diff
@@ -23,10 +31,15 @@ services:
       LUIGI_CORE_DEFAULT-SCHEDULER-URL: http://scheduler:8082
     depends_on:
       - scheduler
+    networks:
+      luigi-net:
+
   db:
-    image: postgres:9.5.4
-    ports:
-      - "5433:5432"
+    image: postgres:12.4
     environment:
       POSTGRES_USER: luigi
+      POSTGRES_PASSWORD: prettybadpassword
```

Comment:
nit: Just use `password` here. It's consistent with other services that use a default postgres password

Reply:
Done.